README.md

```diff
@@ -60,18 +60,21 @@ out when using this package.
     ```go
     import (
         _ "crypto/sha256"
-   	    _ "crypto/sha512"
+        _ "crypto/sha512"
     )
     ```
     This may seem inconvenient but it allows you replace the hash 
     implementations with others, such as https://github.com/stevvooe/resumable.
  
-2. Even though `digest.Digest` may be assemable as a string, _always_ 
+2. Even though `digest.Digest` may be assemblable as a string, _always_ 
     verify your input with `digest.Parse` or use `Digest.Validate`
     when accepting untrusted input. While there are measures to 
     avoid common problems, this will ensure you have valid digests
     in the rest of your application.
 
+3. While alternative encodings of hash values (digests) are possible (for
+    example, base64), this package deals exclusively with hex-encoded digests.
```

Comment:
> The doc update clarifies the point and the interface has been updated to be less opinionated about encoding.

The docs clarify the point that this package exclusively deals with hex encodings?  Can you link that doc?

And while the interface *seems* less opinionated now that there is an [`Algorithm.Encode`][1], that's currently hardcoded to assume hex, `Algorithm` is still not an `interface` (like I proposed in #30), and there is no way for consumers to register or use non-hex algorithms with this package.  If there *is* a way to do that, I'd recommend pushing docs for “how you register your custom non-hex algorithm” to `doc.go`.  With #30, you'd do it by pushing your `Algorithm`-`interface` implementing object into [the `Algorithms` map][2].

>  Other sections look good.

Pulling the typo-fixes and such out (if they still apply) into less contentious PRs might help get them landed faster.

[1]: https://github.com/opencontainers/go-digest/blob/eaa60544f31ccf3b0653b1a118b76d33418ff41b/algorithm.go#L128-L130
[2]: https://github.com/opencontainers/go-digest/pull/30/commits/f97edc8fa724b95596f1575798c8c15dea5993c4#diff-eef025f7a17623416f835581c12f0a3cR96